i18n/ar.json

```diff
@@ -483,7 +483,7 @@
   "confirm_password": "تأكيد كلمة المرور",
   "contain": "محتواة",
   "context": "السياق",
-  "continue": "متابعة",
```

Comment:
Why is this a problem? Seems like it should be a bug with the implementation not something we should have to change.

Reply:
The `slang` tool generates Dart helper classes to provide fully typed translations. It looks like this:

https://github.com/immich-app/immich/blob/37a57d75fd84ec149ee39b64b1a0c6ae1b7aaea3/mobile/lib/i18n/strings_en.g.dart#L197

If I leave the key as "continue", I get the following error when building or running flutter:

```
Error (Xcode): lib/i18n/strings_en.g.dart:197:13: Error:
'continue' can't be used as an identifier because it's a
keyword.
```

The cause is that the generated code contains a getter using a [Dart reserved keyword](https://dart.dev/language/keywords) as a name. I'm not sure how the upstream library would fix this? Open to thoughts here.

Reply:
Id assume the upstream library should not generate invalid code. It should either throw an error or use an alternative naming scheme. It's probably worth opening an issue with them about it. I guess it'd be easy enough to change it either way though.

Reply:
We'll see what they say.

https://github.com/slang-i18n/slang/issues/257

Reply:
@jrasm91 it sounds like this is the expected behavior for the upstream, and it would be a new feature to include a warning that a key is using a reserved keyword. Would you prefer leaving the "continue_text" change out of this PR, until we're more confident about the migration to Slang?

Reply:
Looks like a configuration option to fix this has been added in the latest version of Slang. I reverted the change to `continue`.

Reply:
Awesome!